RadarSDK/RadarLocationManager.m

```diff
@@ -483,6 +483,13 @@ - (void)handleLocation:(CLLocation *)location source:(RadarLocationSource)source
         return;
     }
 
+    BOOL tracking = [RadarSettings tracking];
+    if (!force && !tracking) {
+        [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Skipping location: not tracking"];
```

Comment:
For Android, I know it doesn't have the same shared OS geofences, so we don't have this specific issue - but do locations sometimes get delivered as a side affect of other apps asking for location, even outside of a trackOnce or startTracking?

Reply:
Nope. On Android, you specifically attach a `PendingIntent` to each geofence, and geofence transitions are only sent to your `BroadcastReceiver`. There's also no Android equivalent of significant location changes or visits. No changes needed on Android.